kadok.go

```diff
@@ -158,39 +153,7 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 	}
 
 	if strings.ToUpper(m.Content) == "AKADOK" {
-		mutex.Lock()
-		defer mutex.Unlock()
```

Comment:
Is the mutex used here to enable mutlithreading and making sure that the bot is always able to receive and treat the demands ?

Reply:
Does that mean that several calls to the bot can be made, enabling different processes from the same bot in different channels ?

Reply:
Indeed, the mutex was placed as the bot used only 1buffer to play an audio file. This mutex prevented concurrent executions of the audio feature due to the unique buffer.